Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/adapter/proxy/AdapterPatientDiscoveryProxyWebServiceHelper.java

```diff
@@ -0,0 +1,75 @@
+/**
+ *
+ */
+package gov.hhs.fha.nhinc.patientdiscovery.adapter.proxy;
+
+import gov.hhs.fha.nhinc.adapterpatientdiscoverysecured.AdapterPatientDiscoverySecuredPortType;
+import gov.hhs.fha.nhinc.common.nhinccommon.AssertionType;
+import gov.hhs.fha.nhinc.messaging.client.CONNECTClient;
+import gov.hhs.fha.nhinc.messaging.client.CONNECTClientFactory;
+import gov.hhs.fha.nhinc.messaging.service.port.ServicePortDescriptor;
+import gov.hhs.fha.nhinc.nhinclib.NullChecker;
+import gov.hhs.fha.nhinc.patientdiscovery.PatientDiscoveryException;
+import gov.hhs.fha.nhinc.patientdiscovery.adapter.proxy.service.AdapterPatientDiscoverySecuredServicePortDescriptor;
+import gov.hhs.fha.nhinc.webserviceproxy.WebServiceProxyHelper;
+import org.hl7.v3.PRPAIN201305UV02;
+import org.hl7.v3.PRPAIN201306UV02;
+import org.hl7.v3.RespondingGatewayPRPAIN201305UV02RequestType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author PVenkatakrishnan
+ *
+ */
+public class AdapterPatientDiscoveryProxyWebServiceHelper {
+    private static final Logger LOG = LoggerFactory.getLogger(AdapterPatientDiscoveryProxyWebServiceHelper.class);
+    private static AdapterPatientDiscoveryProxyWebServiceHelper instance = new AdapterPatientDiscoveryProxyWebServiceHelper();
+    private WebServiceProxyHelper oProxyHelper = new WebServiceProxyHelper();
+
+    AdapterPatientDiscoveryProxyWebServiceHelper() {
```

Comment:
follow singleton pattern. add private to prevent others to call new AdapterPatientDiscoveryProxyWebServiceHelper()